pipelines/controller/src/main/resources/templates/index.html

```diff
@@ -135,6 +135,22 @@
       failure.style.display = "none";
     }, 4000);
 
+    function fetchLatestMetrics() {
+      var xhr = new XMLHttpRequest();
+      xhr.open("GET", '/actuator/prometheus', true);
+      xhr.onreadystatechange = function() {
+        if (this.readyState === XMLHttpRequest.DONE && this.status === 200) {
+          metricsBody.style.display = "block";
```

Comment:
Where do we set `metricsBody`?

Reply:
The `innerHTML` property is set in the `metricsBody`, Please refer this [link](https://github.com/google/fhir-data-pipes/blob/1bcc492f2bdc6875b50dc90b46dffa71da94d63d/pipelines/controller/src/main/resources/templates/index.html#L144-L147)
`metricsBody.innerHTML = "<pre>" + this.response + "</pre>";`

Reply:
Agreed, have made the changes to use `document.getElementById()`